sources/main.cpp

```diff
@@ -64,7 +64,7 @@ main(int argc, char* argv[]) {
 
     auto& runtimeConfig = options.config();
     dfl::inputs::Configuration config(runtimeConfig.configPath);
-    dfl::common::Log::init(options, config.outputDir());
+    dfl::common::Log::init(options, config.outputDir(), false);
```

Comment:
Please declare a const bool& append to make the code easier to read

Reply:
OK